Packs/Campaign/Scripts/GetCampaignDuration/GetCampaignDuration.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+import dateutil
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+class FieldNotExists(Exception):
+    pass
+
+
+def get_duration_html():
+    try:
+        incident_id = demisto.incident().get('id', {})
+        context = demisto.executeCommand("getContext", {'id': incident_id})
+        first_date = demisto.get(context[0]['Contents']['context'], "EmailCampaign.firstIncidentDate")
+
+        if not first_date:
+            raise FieldNotExists()
+        if isinstance(first_date, list):
+            first_date = first_date[-1]
+
+        now = datetime.now()
+        first_date = dateutil.parser.parse(first_date)  # type: ignore
+        diff = dateutil.relativedelta.relativedelta(now, first_date)  # type: ignore
```

Comment:
try not to ignore by using typings and if it does not work then ignore only specific errors and not all.

Reply:
done